roles/jd-server/src/lib/mempool/rpc_client.rs

```diff
@@ -47,6 +47,15 @@ impl RpcClient {
             .map(|client| RpcClient { client })
             .map_err(|e| BitcoincoreRpcError::JsonRpc(e.into()))
     }
+    pub fn submit_block(
+        &self,
+        submit_block: SubmitBlock,
+    ) -> Result<HashMap<String, SubmitBlock>, BitcoincoreRpcError> {
```

Comment:
Why we simply don't have a reply that is Result<(), Error>?  most implementations (as well as bitcoin reference) opt for this choice. Anyway, this was specified in bip22, so perhaps it is ok.
https://developer.bitcoin.org/reference/rpc/submitblock.html